Sources/ClientRuntime/Networking/Http/HttpResponse.swift

```diff
@@ -19,7 +20,20 @@ public actor HttpResponse: HttpUrlResponse {
     }
 
     public func setStatusCode(newStatusCode: HttpStatusCode) {
+        let codeBeforeUpdate = self.statusCode.rawValue
         self.statusCode = newStatusCode
+        if newStatusCode.rawValue >= 200 && codeBeforeUpdate < 200 {
+            self.continuation?.resume()
```

Comment:
Since a continuation must be called exactly once, it is a good idea to set it to `nil` after calling `resume` on it.

Reply:
Changed as suggested.